classes/submission/Submission.inc.php

```diff
@@ -257,14 +249,6 @@ function setHideAuthor($hideAuthor) {
 	}
 
 	/**
-	 * Set comments to editor.
-	 * @param $commentsToEditor string
-	 */
-	function setCommentsToEditor($commentsToEditor) {
```

Comment:
The same here -- this is a note for me: 
s. `lib/pkp/plugins/importexport/native/filter/NativeXmlSubmissionFilter.inc.php` and `lib/pkp/classes/submission/SubmissionDAO.inc.php`